site-src/implementation-table.md

```diff
@@ -0,0 +1,25 @@
+The following tables are populated from the conformance reports uploaded by project implementations. They are separated into the extended features that each project supports listed in their reports.
+
+## Gateway Profile
```

Comment:
I'm not sure where/how to fit this in, but we'll likely want to always support the current and previous version of Gateway API here. Maybe we need something like:

```
## v1.1
### Gateway
#### HTTPRoute

...

## v1.0
```

Admittedly a report for v1.1 hasn't merged yet, but that should happen relatively soon with https://github.com/kubernetes-sigs/gateway-api/pull/3025.

Reply:
I think this could also be a followup item though if we can't get it in _this_ PR.

Reply:
yeah i think i'd like to noodle on it a bit. I don't like the way i have the version handled, but maybe a way to have older pages potentially? 🤔